mk/common.mk

```diff
@@ -15,12 +15,12 @@ INSTALL  ?= install
 STRIP    ?= strip
 OBJCPY   ?= objcopy
 RPCGEN   ?= rpcgen
-BMAKE    ?= MAKEFLAGS= bmake
+BMAKE    ?= MAKEFLAGS= make 
 
 DATE     := $(shell date -u --iso-8601=minutes)
 REVISION := $(shell git rev-parse HEAD)
 COMPILER := $(realpath $(shell which $(CC)))
-PLATFORM ?= $(shell uname -p)
+PLATFORM ?= $(shell uname -m)
```

Comment:
uname -p for debian returns "unknown", in case you're wondering why I changed this.